lib/checks/aria/aria-required-children-evaluate.js

```diff
@@ -103,7 +108,7 @@ function ariaRequiredChildrenEvaluate(node, options, virtualNode) {
     return true;
   }
 
-  const ownedRoles = getOwnedRoles(virtualNode);
+  const ownedRoles = getOwnedRoles(virtualNode, required);
```

Comment:
Can you maybe just change like 107 to `if (required === null) {`. Right now required looks like a boolean. It's a little confusing.